app/_layout.tsx

```diff
@@ -95,7 +95,7 @@ export default function RootLayout() {
     <SWRConfig value={swrConfiguration}>
       <ThemeProvider value={isDarkColorScheme ? DARK_THEME : LIGHT_THEME}>
         <StatusBar style={isDarkColorScheme ? "light" : "dark"} />
-        <SafeAreaView className="w-full h-full bg-background">
+        <SafeAreaView className="w-full h-full">
```

Comment:
Did you also check that in dark mode? 

Totally possible we can omit that now because these colors are now used from some other color definition, just wanted to make sure.

Reply:
Oh no, it's actually a typo, I was adding a random color to see where it starts and ends, reverting this.